tests/passes/static-timing/compile-seq-mixed.expect

```diff
@@ -0,0 +1,63 @@
+enables:
+[0, 1)
+  incr_r0[go] = 1'd1;
+[1, 2)
+  incr_r1[go] = 1'd1;
+transitions:
+(0)->(1)
+  1'b1
+(1)->(2)
+  1'b1
```

Comment:
This file should only have transitions and not the Calyx code. The test file should add `-b none` to hide the generated Calyx code.

Reply:
I thought it might be good to test the control program was updated correctly, but if it's not important I can remove it. Is there any way to just test the control program or does runt check the entire output verbatim?